lms/djangoapps/grades/models.py

```diff
@@ -517,7 +517,7 @@ class Meta(object):
 
     # Information about the course grade itself
     percent_grade = models.FloatField(blank=False)
-    letter_grade = models.CharField(u'Letter grade for course', blank=False, max_length=255)
+    letter_grade = models.CharField(u'Letter grade for course', blank=False, null=False, max_length=255)
```

Comment:
I'd thought to include it just to make it clear that we are intentionally preventing null values instead of defaulting to whatever Django uses.